bugbug/bug_features.py

```diff
@@ -481,6 +484,13 @@ def __call__(self, bugs, **kwargs):
         return bugs[0]["target_milestone"] == bugs[1]["target_milestone"]
 
 
+class is_first_affected_same(couple_bug_feature):
+    def __call__(self, bugs, **kwargs):
+        return min(
+            map(int, affected_then_unaffected().get_lists(bugs[0], **kwargs)[1])
```

Comment:
You are creating a new object of class affected_then_unaffected simply to call `get_lists`. It'd be better instead to define get_lists (with a better name, as `get_lists` is *really* generic :P) as a helper function outside of any class, given that it is useful for many classes and it's not directly linked to `affected_then_unaffected`.